packages/contracts-bedrock/contracts/test/invariants/SafeCall.t.sol

```diff
@@ -78,25 +92,43 @@ contract SafeCaller_Actor is StdUtils {
         FAILS = _fails;
     }
 
-    function performSafeCallMinGas(uint64 gas, uint64 minGas) external {
+    function performSafeCallMinGas(
+        uint64 gas,
+        uint64 minGas,
+        address to,
+        uint8 value,
+        uint16 reservedGas
+    ) external {
+        // Only send to EOAs - we exclude the console as it has no code but reverts when called
+        // with a selector that doesn't exist due to the foundry hook.
+        vm.assume(to.code.length == 0 && to != 0x000000000000000000636F6e736F6c652e6c6f67);
```

Comment:
Should be able to access this as `CONSOLE`
https://github.com/foundry-rs/forge-std/blob/fc560fa34fa12a335a50c35d92e55a6628ca467c/src/Base.sol#L11

Reply:
`CONSOLE` is a part of `Base.sol`, and this actor contract only inherits from `StdUtils`.